BUILD.bazel

```diff
@@ -81,3 +84,52 @@ pkg_tar(
     strip_prefix = "./rules_nodejs_package",
     tags = ["manual"],
 )
+
+BAZEL_VERSION = "0.28.1"
+
+BAZEL_BINARY = "@build_bazel_bazel_%s//:bazel_binary" % BAZEL_VERSION.replace(".", "_")
+
+NPM_PACKAGES = {
```

Comment:
my docs change introduces a similar map. At some point we'll need a globally shared variable for this like in package.bzl or something